creates task-1 js and html

Sebs5384 opened the pull request:
this script create inputs and takes it values upon the user spicifications to calculate min, max and avg ages.

Sebs5384 committed:
creates task-1 js and html

this script create inputs and takes it values upon the user spicifications to calculate min, max and avg ages.

fsodano commented on task-1-index.html:
```diff
@@ -0,0 +1,20 @@
+<body>
+  <form id="members-ask" onsubmit="return false">
+    <strong id="family-ask">¿How many members does your family have?</strong><br>
+    <input type="number" placeholder="Entry members" id="member">
```
member no es un buen nombre, members-quantity quizás?

fsodano commented on task-1-index.html:
```diff
@@ -0,0 +1,20 @@
+</head>
+<body>
+  <form id="members-ask" onsubmit="return false">
+    <strong id="family-ask">¿How many members does your family have?</strong><br>
```
family-ask no tiene sentido

fsodano commented on task-1-index.html:
```diff
@@ -0,0 +1,20 @@
+  <title>Clase 6 - Tarea 1</title>
+</head>
+<body>
+  <form id="members-ask" onsubmit="return false">
```
members-ask no tiene sentido, el onsubmit no va acá, debería ir en el JS

fsodano commented on task-1-index.html:
```diff
@@ -0,0 +1,20 @@
+
+  <div id="members-list">
+
+    <strong id="announce"></strong>
```
qué anuncia?

fsodano commented on task-1.js:
```diff
@@ -0,0 +1,89 @@
+document.querySelector("#send-members").onclick = function(){
```
a quién le mandás miembros? qué es mandar un miembro?

fsodano commented on task-1.js:
```diff
@@ -0,0 +1,89 @@
+    if (membersNumber == 0){
+        $announce.innerText = 'Introduce valid numbers !'
+    } else if (!document.querySelector('#new-member')){ 
+     createMembers(membersNumber);
+     createButtons();
+     $announce.remove();
```
mal indentado

fsodano commented on task-1.js:
```diff
@@ -0,0 +1,89 @@
+    }
+}
+
+function createMembers(number){
```
quantity quizás es mejor

fsodano commented on task-1.js:
```diff
@@ -0,0 +1,89 @@
+            $announce.textContent = `Please introduce valid ages for each member`
+            return;
+           }
+           min = Math.min(min, number)
+           max = Math.max(max, number)
```
esto debería ser un algoritmo hecho por vos, para practicar

fsodano commented on task-1.js:
```diff
@@ -0,0 +1,89 @@
+           }
+           min = Math.min(min, number)
+           max = Math.max(max, number)
+           avg += number   
```
no abrevies

fsodano commented on task-1.js:
```diff
@@ -0,0 +1,89 @@
+    $div.appendChild($strong)
+
+    $calculateButton.onclick = function(){
+        const ageValues = document.querySelectorAll('.create-member')
```
lo llamás age values pero en realidad es un input list de edades... y ese input list tiene por clase create-member!? no tiene sentido.

fsodano commented on task-1.js:
```diff
@@ -0,0 +1,89 @@
+        let avg = 0
+        for (let i = 0; i < ageValues.length; i++){
+      
+           let number = (parseInt(ageValues[i].value))
```
age

fsodano commented on task-1.js:
```diff
@@ -0,0 +1,89 @@
+     $announce.appendChild($redo)
+    }
+   
+    $redo.onclick = function(){ 
```
reest, qiuzás? En vez de redo.

fsodano commented on task-1.js:
```diff
@@ -0,0 +1,89 @@
+    }
+   
+    $redo.onclick = function(){ 
+        if(!document.querySelector('#members-list') === false){ 
```
en vez de if ! algo == false podés preguntar if(algo)

Sebs5384 committed:
fixes bad practices

changes some elements following good practices and the maximum,minimum and average functions for IF algorithms

Sebs5384 committed:
updates reset button

$reset was being created as "button" instead of just "reset" making $reset.type and $reset.innerText unnecessary

Sebs5384 committed:
add new files to task-1-class-6

changes task-1.js logic by adding new functions, adds calculations, task-1.css and validations files.

Sebs5384 committed:
add testing file

testing validations as well as find maximum, minimum, average and getMember functions
